pkg/agent/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -73,6 +73,10 @@ type Controller struct {
 	networkPolicyWatcher  *watcher
 	appliedToGroupWatcher *watcher
 	addressGroupWatcher   *watcher
+	appliedToSyncGroup    sync.WaitGroup
+	addressSyncGroup      sync.WaitGroup
+	internalNPSyncGroup   sync.WaitGroup
+	restartSynced         bool
```

Comment:
I can understand why we have a `restartSynced` for each watcher but not why we have this field here for the controller. It seems to never be changed after initialization?

Reply:
Good point. Removed this field.